apps/frontend/src/components/App.tsx

```diff
@@ -43,10 +55,17 @@ export default function App() {
           >
             <MenuIcon />
           </IconButton>
-          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          <Typography
+            variant="h6"
+            component="div"
+            sx={{ flexGrow: 1 }}
+            onClick={routeHome}
+          >
             Habit Helper
           </Typography>
```

Comment:
This can be replaced with:
```jsx
<Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
  <Link
    component={RouterLink}
    to="/"
    color="inherit"
    underline="none"
  >
    Habit Helper
  </Link>
</Typography>
```

In this case RouterLink would be an alias to the React Router Link component ie. 
```jsx
import {
  Route,
  Routes,
  Link as RouterLink,
} from "react-router-dom";
```

This is a little more verbose but is more clear in its intentions and has the added benifit of making the title appear clickable on hover.